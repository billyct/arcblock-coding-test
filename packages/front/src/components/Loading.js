import {ReactComponent as LoadingIcon} from '../assets/loading.svg'

const Loading = () => (
  <div className='w-full py-20 text-center'>
    <LoadingIcon className='animate-spin h-12 w-12 inline-block'/>
  </div>
)

export default Loading
