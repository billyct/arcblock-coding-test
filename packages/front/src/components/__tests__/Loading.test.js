import {render} from '@testing-library/react'

import Loading from '../Loading'

test('Loading Component', () => {
  const {container} = render(<Loading/>)
  expect(container).toMatchSnapshot()
})
