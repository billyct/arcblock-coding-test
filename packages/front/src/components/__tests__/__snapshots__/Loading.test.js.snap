// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component 1`] = `
<div>
  <div
    class="w-full py-20 text-center"
  >
    <svg
      class="animate-spin h-12 w-12 inline-block"
    >
      loading.svg
    </svg>
  </div>
</div>
`;
